### DISCUSSION TITLE
Feature find or request: report multiple datas at one table

### DISCUSSION THREAD

**Discussion by hermitwhite** [OpenChrom/openchrom#480] [Q&A]
It's good to have a function to report datas in multiple files as a single table to paste into spreadsheet programs, and I did not find it yet.

I found that "Template Chromatogram Report" with "Append" option could report datas within a file,  but it seems there are no option to remove the header and the tags, which made it could not become a table. Maybe a "report in single line" or "data only" option works directly. If there are still no function like this.

**Comment by eselmeister** [comment]
A chromatogram could contain reference(s). They are either loaded directly via the converter (Agilent ->MSD & FID) or added manually. Chromatograms in open editors can be added:

![Bildschirmfoto von 2020-10-03 07-38-40](https://user-images.githubusercontent.com/686671/94984539-4f8a0c80-054d-11eb-8b40-403263b38859.png)

The peak template report should match your needs. It can be created via the preferences editor. The editor allows an option to export the data to text format.

![Bildschirmfoto von 2020-10-03 07-38-45](https://user-images.githubusercontent.com/686671/94984543-57e24780-054d-11eb-9afd-0f0b0d7c7aac.png)

When doing a report, run it from the context menu. First try it with all column. Afterwards, you can specify which columns to print:

![Bildschirmfoto von 2020-10-03 07-38-57](https://user-images.githubusercontent.com/686671/94984551-76484300-054d-11eb-8f81-b1f94d01ed0b.png)

**Comment by hermitwhite** [comment]
Thank you. Is it possible to report quantification results in this way? and could I disable the tags of columns and the extra line break, only get a column of numbers?

![Example](https://user-images.githubusercontent.com/5426911/95255207-9597e680-0853-11eb-8b1e-0350a0bc0625.png)

**Comment by Mailaender** [comment]
The `Chromatogram Peak Report (*.csv)` can report quantitation results:
![screenshot](https://github.com/OpenChrom/openchrom/assets/756669/1397b3aa-e16e-4a9e-9842-121287bbbb33)
